feature/experimental/system/gnmi/benchmarking/ate_tests/drained_configuration_convergence_time/drained_configuration_convergence_time_bgp_test.go

```diff
@@ -252,8 +264,9 @@ func TestEstablish(t *testing.T) {
 
 	t.Log("Build Benchmarking BGP and ISIS test configs.")
 	dutBenchmarkConfig := setup.BuildBenchmarkingConfig(t)
-
-	fptest.LogQuery(t, "Benchmarking configs to configure on DUT", dutConfigPath.Config(), dutBenchmarkConfig)
+	if !*deviations.ExplicitInterfaceInDefaultVRF {
```

Comment:
Why this is only applied to !*deviations.ExplicitInterfaceInDefaultVRF case?

Reply:
The `LogQuery` is expecting interface-id in network-instance to be like `ethernet-8/57`, however due to deviation `ExplicitInterfaceInDefaultVRF` we added the interface-id along with subinterface in network-instance as `ethernet-8/57.0`. Following log print error is observed otherwise - 
```
drained_configuration_convergence_time_bgp_test.go:267: Config for Benchmarking configs to configure on DUT at / render error: validation err: /root/network-instances/network-instance: key field Id: element key ethernet-8/57.0 != map key ethernet-8/57 
/root/network-instances/network-instance: key field Id: element key ethernet-8/58.0 != map key ethernet-8/58
```